Scripts/symbolic_hermitians/generate_code.py

```diff
@@ -247,11 +258,39 @@ def delete_generated_files():
     # Evolve.cpp_compute_dt_fill #
     #============================#
     code = []
-    for t in tails:
-        for i in range(args.N):
-            line = "N_diag_max = max(N_diag_max, state.max(GIdx::N"+str(i)+str(i)+"_Re"+t+"));"
-            code.append(line)
-    code.append("N_diag_max *= 2*"+str(args.N)+";") # overestimate of net neutrino+antineutrino number density
+    length = sympy.symbols("length",real=True)
+    cell_volume = sympy.symbols("cell_volume",real=True)
+    rho = sympy.symbols("fab(i\,j\,k\,GIdx\:\:rho)",real=True)
+    Ye = sympy.symbols("fab(i\,j\,k\,GIdx\:\:Ye)",real=True)
+    mp = sympy.symbols("PhysConst\:\:Mp",real=True)
+    sqrt2GF = sympy.symbols("M_SQRT2*PhysConst\:\:GF",real=True)
```

Comment:
I really like this approach of making symbols out of variables in the C++ scope :)

Reply:
Yeah, there is some other python code that I want to clean up now, but that will wait.